src/docker-osx-dev

```diff
@@ -971,10 +1032,10 @@ function watch_and_sync {
 function install {
   log_info "Starting install of docker-osx-dev"
   install_dependencies
-  init_boot2docker
-  install_rsync_on_boot2docker
+  init_docker_host
+  install_rsync_on_docker_host
   add_docker_host
-  add_environment_variables
+  [ -z "$DOCKER_MACHINE_NAME" ] && add_boot2docker_environment_variables
```

Comment:
Probably better to create a function called `add_environment_variables` that takes care of the `$DOCKER_MACHINE_NAME` check itself.